TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -418,6 +428,8 @@ void runReplayWithIOStreams(Duration observedPacketConnectionTimeout,
                         new TrafficReplayerAccumulationCallbacks(replayEngine, resultTupleConsumer));
         try {
             pullCaptureFromSourceToAccumulator(trafficChunkStream, trafficToHttpTransactionAccumulator);
+        } catch (InterruptedException ex) {
+            throw ex;
```

Comment:
Do we need to add a `Thread.currentThread().interrupt();` here?

Reply:
No.  The general rule is if you're suppressing that this was an interruption, you should reset the interrupt state.  Since we're propagating (so that we don't obscure it), the final catch-receiver can handle re-interrupting.